packages/path_provider/path_provider_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.1.0
```

Comment:
I'd make this 1.0.1, really. In this case, I'd equate it to "fixing a typo" (_in the name of an enum, ahem..._)